news/2022-08-01-osumonthly-1.md

```diff
@@ -0,0 +1,247 @@
+
+<iframe width="100%" height="315" src="https://youtu.be/embed/1RuyJ3y_ot4" frameborder="0" allowfullscreen></iframe>
+
+#### Millions of Long Notes
+
+Wow. Just completely shocked at this performance by [[GS]Myuka](https://osu.ppy.sh/users/10072733) on [Camellia’s - MI11I0N PP](https://osu.ppy.sh/beatmapsets/1388481#mania/2867552). 
+
+If you thought *My Man* had LNs, wait until you see a map of over 7 minutes of non-stop LN gameplay, and with not just single-note rainbows and glissandos, but an interconnected web of the two. It is an impeccable display of skill, accuracy, speed, and just what osu!monthly is looking for in a showcase!
+
+<iframe width="100%" height="315" src="https://youtu.be/embed/nc_ejGtZKUM" frameborder="0" allowfullscreen></iframe>
+
+### ![](/wiki/shared/mode/catch.png) osu!catch
+
+#### Galaxy Brain
+
+Somebody scream! Even with being the number one osu!catch player in the world, a play such as [YesMyDarkness’](https://osu.ppy.sh/users/4158549/fruits) is no small feat. 
+
+Coming in with a HD HR 4xmiss on [Kurokotei’s Galaxy Collapse](https://osu.ppy.sh/beatmapsets/521900#fruits/1108507) one of the most well known artists and songs in osu! surely does do them some justice. Riddled with intense and **dash**ingly hard jumps it is a play to contend with even the hardest of maps, let alone with hidden and hardrock! 
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/byyHOAopo1A" frameborder="0" allowfullscreen></iframe>
+
+---
+
+## New Features on osu!(lazer)!
+
+The new client has never so **close** from releasing (there's still a lot of work on it!), and has **new mods**, more original than never! 
+For the first time ever, lazer patch notes have been posted on the official osu! [YouTube channel](https://www.youtube.com/c/osugame), compiled by [pishifat](https://osu.ppy.sh/users/3178418) with further insight provided by the man himself, [peppy](https://osu.ppy.sh/users/2)!
+Check the video out:
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/67P4LKlAh-0" frameborder="0" allowfullscreen></iframe>
+
+If you are interested in all previous and future patch notes, you can find them [here](https://osu.ppy.sh/home/changelog).
+
+One thing you might've missed is the new fancy Repel mod added to lazer, which is pretty chaotic but fun (finally a worthy opponent for [badeu](https://osu.ppy.sh/users/1473890)?)
+
+<div align="center">
+    <video width="100%" controls>
+        <source src="https://user-images.githubusercontent.com/47010459/172435116-6b09a48c-6fd6-4abb-9e5f-e96ab3565184.mp4" type="video/mp4" preload="none">
```

Comment:
these need to be static hosted on assets.ppy.sh 100%, don't trust this link to stay active forever

will do this before release

Reply:
Oh, we were wondering about this, thanks for telling!

Reply:
If it helps the video on line 173 was ripped from this [patch notes post](https://osu.ppy.sh/home/changelog/lazer/2021.514.0) but I couldn't find where the patch notes are held on Github so